### PR TITLE
make changes to create all required nupkgs for perf tests

### PR DESCRIPTION
**Pull request by rohit21agrawal**
this is an engineering fix to create nupkgs out of all required projects that would be required for setting up the perf lab infrastructure for nuget.

### CODE REVIEW

**Review by jainaashish — APPROVED** [review]
LGTM

File: src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj [hunk 1]
```diff
@@ -480,4 +482,10 @@
       <SubType>Designer</SubType>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="NuGet.Build.Tasks.Pack" Version="4.4.0" PrivateAssets="all"/>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj:486]: Hacks :P

File: src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj [hunk 2]
```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <Shipping>true</Shipping>
     <IncludeInVSIX>true</IncludeInVSIX>
+    <PackProject>true</PackProject>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj:8]: nit: why are the changes in 2 different places :D compare this to nuget.packagemanager.ui.csproj

**Review by nkolev92 — APPROVED** [review]

File: src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj (hunk 2 above)
Comment (rohit21agrawal) [reply src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj:8]: any project that has to be packed has to explicitly set this. it defaults to false.

Comment (nkolev92) [reply src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj:8]: nah, my question/nit was the formatting :)
Half the projects had the new projects one under the other line, the others didn't :)

**Pull request merged**